tests/dragonfly/generic_test.py

```diff
@@ -83,3 +83,26 @@ async def task2(k, n):
     await asyncio.gather(
         task1("i1", 2), task1("i2", 3), task2("l1", 2), task2("l1", 2), task2("l1", 5)
     )
+
+
+@dfly_args({"port": 6377})
+async def test_arg_from_environ_overwritten_by_cli(df_local_factory):
+    os.environ["DFLY_port"] = "6378"
+    dfly = df_local_factory.create()
+    dfly.start()
+    client = aioredis.Redis(port="6377")
+    await client.ping()
+
+
+async def test_arg_from_environ(df_local_factory):
```

Comment:
Can you also add a test for nonexisting flag env var?

Reply:
Already requested that as well 😆